azkaban-web-server/src/web/js/azkaban/view/schedule-sla.js

```diff
@@ -189,6 +189,14 @@ azkaban.ChangeSlaView = Backbone.View.extend({
       var duration = rFlowRule.cells[2].firstChild.value;
       var email = rFlowRule.cells[3].firstChild.checked;
       var kill = rFlowRule.cells[4].firstChild.checked;
+      if (email === false && kill === false) {
+        alert("Unable to create SLA Action. Please set either Kill or Email"
+            + " action");
```

Comment:
typo: _actions_, in plural I believe.

Reply:
I don't think it should be plural here as this code will generate a separate alert per bad entry. Is that right Deepak?

Reply:
Please consider rephrasing as: "Unable to create SLA Action. Please check at least one action."
(because is possible to check both actions)

Reply:
Yes. This is running in a for loop for each SLA action.